cmd/cephcsi.go

```diff
@@ -180,11 +187,29 @@ func main() {
 
 	case livenessType:
 		liveness.Run(&conf)
+
+	case controllerType:
+		cfg := controller.Config{
+			DriverName: dname,
+			Namespace:  conf.DriverNamespace,
+		}
+		// initialize all controllers before starting.
+		initControllers()
+		err = controller.Start(cfg)
+		if err != nil {
+			logAndExit(err.Error())
+		}
 	}
 
 	os.Exit(0)
 }
 
+// initControllers will initialize all the controllers.
+func initControllers() {
+	// Add list of controller here.
+	persistentvolume.Init()
```

Comment:
If you call the function `init()`, you do not need to call it explicitly here, just importing the package should be sufficient.

Reply:
done it explicitly as this should not be called for cephfs or node servers.